README.md

```diff
@@ -76,39 +76,40 @@ Examples of property usage can be found in the test project in the following pla
-| region   | `eu-west-2` | The AWS region where the queues live. |
-| localstackUrl | `http://localhost:4566` | Only used for `provider=localstack`. The location of the running LocalStack instance. |
-| queues   | | A map of `queueId` to `QueueConfig`. One entry is required for each queue. In production these are derived from environment variables with the prefix `HMPPS_SQS_QUEUES_` that should be populated from Kubernetes secrets (see below).
-| topics   | | A map of `topicId` to `TopicConfig`. One entry is required for each topic. In production these are derived from environment variables with the prefix `HMPPS_SQS_TOPICS_` that should be populated from Kubernetes secrets (see below).
+| Property      | Default                 | Description                                                                                                                                                                                                                             |
+|---------------|-------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| provider      | `aws`                   | `aws` for production or `localstack` for running locally / integration tests.                                                                                                                                                           |
+| region        | `eu-west-2`             | The AWS region where the queues live.                                                                                                                                                                                                   |
+| localstackUrl | `http://localhost:4566` | Only used for `provider=localstack`. The location of the running LocalStack instance.                                                                                                                                                   |
+| queues        |                         | A map of `queueId` to `QueueConfig`. One entry is required for each queue. In production these are derived from environment variables with the prefix `HMPPS_SQS_QUEUES_` that should be populated from Kubernetes secrets (see below). |
+| topics        |                         | A map of `topicId` to `TopicConfig`. One entry is required for each topic. In production these are derived from environment variables with the prefix `HMPPS_SQS_TOPICS_` that should be populated from Kubernetes secrets (see below). |
+| reactiveApi   | `false`                 | Publishes a reactive API for the Queue Admin Endpoints. See [Reactive Queue Admin Endpoints](#reactive-queue-admin-endpoints) for more details.                                                                                         |
```

Comment:
This is the only real change to this table - Intellij got carrier away and reformatted everything (though it does look prettier now)